examples/MCP4921_wave_generator_RP2040/.arduino-ci.yml

```diff
@@ -0,0 +1,12 @@
+compile:
+  # Choosing to run compilation tests on 2 different Arduino platforms
+  platforms:
+    # - uno
+    # - due
+    # - zero
+    # - leonardo
+    # - m4
+    # - esp32
+    # - esp8266
+    # - mega2560
+    - ARDUINO_ARCH_RP2040
```

Comment:
RP2040 is not supported in the Arduino build-CI

Created an issue here - https://github.com/Arduino-CI/arduino_ci/issues/329